diann/1.8.0/Dockerfile

```diff
@@ -0,0 +1,36 @@
+################## BASE IMAGE ######################
+FROM ubuntu:latest
+
+################## METADATA ######################
+LABEL base_image="ubuntu:latest"
+LABEL version="1"
+LABEL software="diann"
+LABEL software.version="1.8.0"
+LABEL about.summary="DIA-NN - a universal software for data-independent acquisition (DIA) proteomics data processing."
+LABEL about.home="https://github.com/vdemichev/DiaNN"
+LABEL about.documentation="https://github.com/vdemichev/DiaNN"
+LABEL about.license_file="https://github.com/vdemichev/DiaNN/LICENSE.txt"
+LABEL about.license="SPDX:CC-BY-NC-4.0"
+LABEL about.tags="Proteomics"
+
+################## MAINTAINER ######################
+MAINTAINER Yasset Perez-Riverol <ypriverol@gmail.com>
+
+################## INSTALLATION ######################
+
+ENV DEBIAN_FRONTEND noninteractive
+
+## Update and install packages
+RUN apt-get update -y && apt-get install -y git g++ build-essential cmake zlib1g-dev libbz2-dev libboost-all-dev wget
```

Comment:
why installing all of this, will hightly increase image size and not needed as installing a deb

Reply:
furthermore many are -dev , so include extras only needed for compilation...